MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -464,6 +464,11 @@
 				portabilityFeatures->vertexAttributeAccessBeyondStride = true;	// Costs additional buffers. Should make configuration switch.
 				break;
 			}
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_EXPECT_ASSUME_FEATURES: {
+				auto* shaderExpectAssume = (VkPhysicalDeviceShaderExpectAssumeFeatures*)next;
+				shaderExpectAssume->shaderExpectAssume = true;
+				break;
+			}
```

Comment:
`ShaderExpectAssume` also needs to be added to `MVKDeviceFeatureStructs.def` (again, consistent order).